opentracing/context.go

```diff
@@ -0,0 +1,39 @@
+package opentracing
```

Comment:
("DCP" is too acronymy... I don't have strong feelings about this particular naming possibility.)